### ISSUE TITLE
Update to platform-tools-31.0.2

### ISSUE THREAD

**Issue by chirayudesai** [AOSPAlliance/device-flasher#15]
https://developer.android.com/studio/releases/platform-tools#3102_april_2021

It'll allow us to remove the jasmine_sprout hacks.

It's not up yet (not the tags, nor the binaries), but once it we can switch.

https://dl.google.com/android/repository/platform-tools_r31.0.2-linux.zip

**Comment by Uldiniad** [comment]
done?

**Comment by chirayudesai** [comment]
> done?

No, we didn't merge / ship that yet, and it still needs to be done for the latest versions.
